frontend/app/src/people/utils/OrgNoResults.tsx

```diff
@@ -3,9 +3,25 @@ import { observer } from 'mobx-react-lite';
 import { widgetConfigs } from './Constants';
 import NoneSpace from './NoneSpace';
 
-function OrgNoResults() {
+function OrgNoResults(props: { showAction: boolean, action: () => void }) {
   const tabs = widgetConfigs;
 
+  if (props.showAction) {
+    return (
+      <NoneSpace
+        small
+        style={{
+          margin: 'auto',
+          marginTop: '10%'
+        }}
```

Comment:
looks like the nonespace below uses the same style you can make this a styled component